### PR TITLE
Crerate job to run linters

### LINKED ISSUE #11: CI upgrade
Now CI is quite simple. We could add more checks e.g. black, isort, pylint.

### PR DESCRIPTION
**Pull request by gkodrzycki**
Close #11

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches:
       - main
-      - 6-docker-compose-instead-of-docker-compose
+      - '[0-9]*-*[a-zA-Z]*'  # regex should match typical issue branches names
```
Comment (Kabanosk) [inline .github/workflows/test.yml:7]: I thing better branch naming would be `feature/*` for each issue branch

```suggestion
      - 'feature/*'
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -63,3 +63,32 @@ jobs:
...
+        run: |
+          poetry run isort --check-only .
+
+      - name: Run black
+        run: |
+          poetry run black --check .
+
+      - name: Run pylint
+        run: |
+          poetry run pylint **/*.py
```
Comment (Kabanosk) [inline .github/workflows/test.yml:94]: Not sure if we want linting in workflow named `test.yml`. Maybe we can create new workflow for linting?

File: tests/test_database.py [hunk 3]
```diff
@@ -1,5 +1,3 @@
-import pytest
```
Comment (Kabanosk) [inline tests/test_database.py:1]: Aren't this necessary?

**Review by Kabanosk — CHANGES_REQUESTED** [review]

File: tests/test_database.py (hunk 3 above)
Comment (gkodrzycki) [reply tests/test_database.py:1]: Not at all, even pylint suggests that it is redundant

**Pull request merged**